notebooks/250-efficient-object-detection/250-efficient-object-detection.ipynb

```diff
@@ -0,0 +1,622 @@
+{
```

Comment:
<p>&gt;To test model inference, create an inference pipeline with the help of the above-defined utility functions\  The pipeline consists of :- </p><p>I do not know, it is issue with my viewer, I see some strange symbols in this line (\ and -). Could you please remove?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>